Makefile

```diff
@@ -9,13 +9,14 @@ TESTS=$(wildcard tests/test_*.c);
 OBJCOPY=objcopy
 VERSION=0.1
 
-PREFIX = /usr/local
 
+PREFIX ? = /usr/local
+DESTDIR =
 DIST_DIR = $(MINIPRO)-$(VERSION)
-BIN_DIR = $(PREFIX)/bin/
-UDEV_RULES_DIR = /etc/udev/rules.d/
-MAN_DIR = $(PREFIX)/share/man/man1/
-COMPLETIONS_DIR = /etc/bash_completion.d/
+BIN_DIR = $(DESTDIR)$(PREFIX)/bin/
```

Comment:
Seems like this was already fixed and DESTDIR is present in the current makefile.